src/braille/internal/braille.h

```diff
@@ -55,6 +57,36 @@ class TimeSig;
 class Tuplet;
 class Volta;
 
+class BrailleEngravingItems
+{
+public:
+    BrailleEngravingItems();
+    ~BrailleEngravingItems();
+
+    void clear();
+
+    void join(BrailleEngravingItems*, bool newline = true, bool del = true);
+    void join(std::vector<BrailleEngravingItems*>, bool newline = true, bool del = true);
+
+    QString brailleStr();
+    std::vector<std::pair<EngravingItem*, std::pair<int, int> > >* items();
```

Comment:
It might be helpful to add a small comment somewhere about what the `std::pair<int, int>` means.